apps/adservice/values.yaml

```diff
@@ -15,9 +15,9 @@
 appname: adservice
```

Comment:
Your code must support multi-stage deployment, you can use 2 pieces of values.yaml files